README.md

```diff
@@ -3,10 +3,10 @@
 [![PyPI version](https://badge.fury.io/py/aiida-cp2k.svg)](https://badge.fury.io/py/aiida-cp2k)
 
 # AiiDA CP2K
-**The CP2K plugin for the AiiDA workflow and provenance engine.**
+[AiiDA](http://www.aiida.net/) plugin for [CP2K](https://www.cp2k.org/).
 
-
-![logo](./aiida-cp2k_logo.png)
+# Documentation
```

Comment:
probably use ## everywhere except for the top one

Reply:
Yep, I agree that it looks better.